scripts/insert_into_appointments.sql

```diff
@@ -1 +1,37 @@
 -- insert appointments creating connections between doctors and patients
+
+INSERT INTO appointments(start_at, length, patient_id, doctor_id)
```

Comment:
None of these `INSERT` statements will work because you're trying to insert three data values into four columns.